setup.cfg

```diff
@@ -14,3 +14,15 @@ ignore =
     E203,
     D202,
     W504
+
+[mypy]
+python_version = 3.8
+
+[mypy-glocaltokens.*]
+ignore_missing_imports = True
+
+[mypy-homeassistant.*]
+ignore_missing_imports = True
+
+[mypy-voluptuous.*]
+ignore_missing_imports = True
```

Comment:
Why is it needed?

Reply:
These packages don't distrubute typing information (don't have `py.typed` marker) and `mypy` will complain that it can't import these packages.

Reply:
Got it, thank you!
Check `glocaltokens` open PR, i think it's ready to go. I have added `py.typed` as requested :)

Reply:
Yes, with it we can remove `glocaltokens` from here and type check its usages :)